ServerCore/Pages/Submissions/Index.cshtml

```diff
@@ -101,18 +104,25 @@
         {
             if (Model.DuplicateSubmission)
             {
-            <div class="alert alert-danger" role="alert">
-                <h4>Oops</h4>
-                <span>You've already submitted @Model.SubmissionText!</span>
-            </div>
+                <div class="alert alert-danger" role="alert">
+                    <h4>Oops</h4>
+                    <span>You've already submitted "@Model.SubmissionText!"</span>
+                </div>
+            }
+
+            if (!string.IsNullOrEmpty(Model.AnswerAlertMessage))
```

Comment:
I know I opened that bug about Incorrect, but ideally we'd also put the partial responses here too if they exist.